tests/rabota_by_tests.py

```diff
@@ -0,0 +1,38 @@
+import math
+
+from models.parser_jobs_tut_by import RabotaByParser
+
+
+def test_connection(get_response):
+    assert get_response.status_code == 200, 'Connection error'
+
+
+def test_shotgun_not_found(get_query_shotgun):
+    assert 'По запросу «shotgun» ничего не найдено' in get_query_shotgun.text, 'Query give responses'
```

Comment:
How can you understand what has happened from 'Query give responses' assert message&

Reply:
Fixed